cfme/containers/container.py

```diff
@@ -58,3 +40,67 @@ def get_detail(self, *ident):
         """
         self.load_details(refresh=True)
         return details_page.infoblock.text(*ident)
+
+
+@navigator.register(Container, 'All')
+class ContainerAll(CFMENavigateStep):
+    prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
+
+    def am_i_here(self):
+        return match_page(summary='All Containers')
+
+    def step(self):
+        from cfme.web_ui.menu import nav
+        nav._nav_to_fn('Compute', 'Containers', 'Containers')(None)
+
+    def resetter(self):
+        tb.select('Grid View')
+        if paginator.page_controls_exist():
+            sel.check(paginator.check_all())
+            sel.uncheck(paginator.check_all())
+
+
+@navigator.register(Container, 'Details')
+class ContainerDetails(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    'Container "jenkins" (Summary)'
+    def am_i_here(self):
+        return match_page(summary='Container "{}" (Summary)'.format(self.obj.name))
+
+    def step(self):
+        tb.select('List View')
+        list_tbl.click_row_by_cells({'Name': self.obj.name})
+
+
+@navigator.register(Container, 'EditTags')
+class ContainerEditTags(CFMENavigateStep):
+    prerequisite = NavigateToSibling('Details')
+
+    def am_i_here(self):
+        return match_page(summary='Container "{}" (Summary)'.format(self.obj.name))
```

Comment:
The am_i_here() summaries are the same for EditTags, Timelines, and Utilization. They need to be unique or not used at all.